tests/test_helpers.nim

```diff
@@ -0,0 +1,91 @@
+import os, osproc, strutils, streams, pegs
+
+type
+  CompileInfo* = object
+    templFile*: string
+    errorFile*: string
+    errorLine*, errorColumn*: int
```

Comment:
`nim check` is able to return multiple errors on each invocation. This module will be even more helpful if they can collect all of them.

Reply:
I didn't know about the nim check command, interesting.

Perhaps it would be better to run the check command instead of compile when a test is run specifically for compile error output (and thus meant to fail on compile). It will also be a bit faster as check does not generate code.